shacl/dataset.jsonld

```diff
@@ -70,13 +79,25 @@
           "sh:path": {
             "@id": "schema:potentialAction"
           },
-          "sh:minCount": 1,
-          "sh:class": {
-            "@id": "schema:SearchAction"
+          "sh:qualifiedValueShape": {
+            "sh:class": {
+              "@id": "schema:SearchAction"
+            }
```

Comment:
I _could_ add `"sh:node": "schema:SearchActionShape"` here (and give the shape below that IRI) but it isn’t necessary: SHACL will just match _any_ `schema:SearchAction`s for us and ensure that each has a `schema:query`, which is fine.

Reply:
👍🏻